show_char_code.py

```diff
@@ -11,4 +11,4 @@ def on_activated(self, view):
 
     def show_char_code(self, view):
         selected = view.substr(view.sel()[0].a)
-        view.set_status('charcode', "Char Code: {0}".format(ord(selected)))
```

Comment:
I liked the cleaner look, but I agree that #x is a better approach.

I saw the fork doing something similar. I didn't want to clutter up the bottom too much and thought binary might be a bit excessive. Maybe making it an option that can be turned on/off?